config/environments/development.rb

```diff
@@ -51,4 +51,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Search options for member
+  config.advanced_search = false
```

Comment:
`algolia_search` might be a better option here than `advanced_search`, as there might possibly be other _advanced search_ backends supported in the future. You might also consider moving this configuration to an initializer file to reduce duplication and consolidate the configuration. This would prevent churn of Rails-generated files and allow usage like:

```ruby
# Enable everywhere but the `test` environment
Rails.application.config.algolia_search = !Rails.env.test?

# or

# Enable in the `staging` and `production` environments
Rails.application.config.algolia_search = Rails.env.in?(%w[staging production])
```

Reply:
Thanks!  Updated configuration name to algolia_search and created a [new issue](https://github.com/renocollective/member-portal/issues/93) for adding configuration initializer.